spanner/transaction.go

```diff
@@ -825,6 +825,9 @@ func (t *ReadOnlyTransaction) acquireMultiUse(ctx context.Context) (*sessionHand
 				},
 			}
 			t.mu.Unlock()
+			if sh != nil {
+				sh.updateLastUseTime()
```

Comment:
It seems that we are only setting this property for multi-use transactions. I think that we should also set it for single-use transactions.
Also, it makes more sense to set this property when an RPC is actually invoked, as that is the moment that the session is used according to the server. So instead of calling this method in the various transaction methods, it should be called whenever the Read, ExecuteSql, ExecuteStreamingSql etc. RPCs are invoked.

Reply:
Updated code to set lastUseTime property before RPC invocation. Can you please take a look?